[Bug]: Maven repo down

### `/mv version -p` output

NA

### Server logs

NA

### Server Version

```shell
NA
```


### Bug Description

https://repo.onarandombox.com/content/groups/public/com/onarandombox/multiversecore/Multiverse-Core/4.3.8/

Seems the repo is down. Can anything be done?

Thanks.

### Steps to reproduce

NA

### Agreements

- [X] I have searched for and ensured there isn't already an open or resolved issue(s) regarding this.
- [X] I was able to reproduce my issue on a freshly setup and up-to-date server with the latest version of Multiverse plugins with no other plugins and with no kinds of other server or client mods.

Thanks for the quick response.

```
[ERROR] Failed to execute goal on project (redacted): Could not resolve dependencies for project (redacted): Could not find artifact com.onarandombox.multiversecore:multiverse-core:jar:4.3.8 in OnARandomBox (https://repo.onarandombox.com/content/groups/public/) -> [Help 1]

```
<repositories>
  <repository>
      <id>minebench-repo</id>
      <url>https://repo.minebench.de/</url>
  </repository>

  <repository>
      <id>OnARandomBox</id>
      <url>https://repo.onarandombox.com/content/groups/public/</url>
  </repository>
...
```

```    
<dependencies>
  <dependency>
      <groupId>de.themoep.idconverter</groupId>
      <artifactId>mappings</artifactId>
      <version>1.2-SNAPSHOT</version>
  </dependency>

  <dependency>
      <groupId>com.onarandombox.multiversecore</groupId> <!-- Don't forget to replace this -->
      <artifactId>multiverse-core</artifactId> <!-- Replace this as well -->
      <version>4.3.8</version> <!-- This too -->
      <scope>provided</scope>
  </dependency>
...
```

idconverter also does not resolve, however, adding their repo fixes that and it is not your issue.

With the dependency type set to `all`:

```
<dependency>
    <groupId>com.onarandombox.multiversecore</groupId> <!-- Don't forget to replace this -->
    <artifactId>multiverse-core</artifactId> <!-- Replace this as well -->
    <version>4.3.8</version> <!-- This too -->
    <scope>provided</scope>
    <type>all</type>
</dependency>
```

```
[ERROR] Failed to execute goal on project (redacted): Could not resolve dependencies for project (redacted): Could not find artifact com.onarandombox.multiversecore:multiverse-core:all:4.3.8 in OnARandomBox (https://repo.onarandombox.com/content/groups/public/) -> [Help 1]
```

Strange, I can get by with a system dependency for now.

Just FYI: I don't seem to be the only one https://www.spigotmc.org/threads/cant-get-multiverse-core-to-work.598814/

Thanks for the help.

> if you manually enter this URL into your web browser, what do you get? https://repo.onarandombox.com/content/groups/public/com/onarandombox/multiversecore/multiverse-core/4.3.8/

https://imagedelivery.net/H0ZxB-euirrsPPuyZWi9xQ/d9bc651c-85fe-49b4-9e4f-20c07d2fee00/public

I tried again with the repo, same issue:
```
[ERROR] Failed to execute goal on project (redacted): Could not resolve dependencies for project (redacted): Could not find artifact com.onarandombox.multiversecore:multiverse-core:all:4.3.8 in OnARandomBox (https://repo.onarandombox.com/content/groups/public/) -> [Help 1]
```

With Maven debug enabled:

```
[DEBUG] Using transporter WagonTransporter with priority -1.0 for https://repo.onarandombox.com/content/groups/public/
[DEBUG] Using connector BasicRepositoryConnector with priority 0.0 for https://repo.onarandombox.com/content/groups/public/
Downloading from OnARandomBox: https://repo.onarandombox.com/content/groups/public/com/onarandombox/multiversecore/multiverse-core/4.3.8/multiverse-core-4.3.8.jar
[DEBUG] Writing tracking file /Users/matt/.m2/repository/com/onarandombox/multiversecore/multiverse-core/4.3.8/multiverse-core-4.3.8.jar.lastUpdated
```

With Maven debug with the dependency type set too `all`:

```
[DEBUG] Using transporter WagonTransporter with priority -1.0 for https://repo.onarandombox.com/content/groups/public/
[DEBUG] Using connector BasicRepositoryConnector with priority 0.0 for https://repo.onarandombox.com/content/groups/public/
Downloading from OnARandomBox: https://repo.onarandombox.com/content/groups/public/com/onarandombox/multiversecore/multiverse-core/4.3.8/multiverse-core-4.3.8.all
[DEBUG] Writing tracking file /Users/matt/.m2/repository/com/onarandombox/multiversecore/multiverse-core/4.3.8/multiverse-core-4.3.8.all.lastUpdated
```

Happy to share full POM via Discord (Matt#5727) or email (matt@matthogan.co.uk) if that helps.

Thanks.

--- a/build.gradle
+++ b/build.gradle
@@ -171,6 +171,7 @@ shadowJar {
     configurations = [project.configurations.api]
 
     archiveFileName = "$baseName-$version.$extension"
+    classifier = ''
 }
 
 build.dependsOn shadowJar

